pandas/core/reshape/tile.py

```diff
@@ -396,11 +398,18 @@ def _bins_to_cuts(
             labels = _format_labels(
                 bins, precision, right=right, include_lowest=include_lowest, dtype=dtype
             )
-        else:
+        elif labels:
```

Comment:
i think should be `elif labels is True`?

Reply:
ah yes, thanks for that.